src/CubedSpheres/cubed_sphere_halo_filling.jl

```diff
@@ -143,5 +143,48 @@ function fill_horizontal_velocity_halos!(u::CubedSphereField, v::CubedSphereFiel
         end
     end
 
+    # Manually handle corners to exactly match MITgcm halo filling algorithm.
+    # Not sure whether every single one of these is needed though.
+
+    Nx, Ny, Nz, Nf = size(u.grid)
+
+    @inbounds begin
```

Comment:
Some comments about the local geometry (boundary and rotation of data source relative to destination) could help when prototyping an adjacency function in the future.